docker/docker-entrypoint.sh

```diff
@@ -6,7 +6,4 @@
 # The above will not source bashrc without entrypoint.
 source ~/.bashrc
 
-# Activate pytorch conda env at entry by default.
-conda activate pytorch
```

Comment:
Why are we getting rid of this?

Reply:
We put it within the `.bashrc` which we source above?

Reply:
https://github.com/pytorch/xla/blob/950e12a66fcb8c4e4e4be3b3d280c65cffc978e1/docker/Dockerfile#L27

Reply:
IIRC, that one only was effective for when you run `docker run -it` as an interactive shell as it sources `~/.bashrc`. But for when running as command `docker run ... $IMAGE python test.py` then it didn't source the rc and instead consumed entrypoint. So I think we need both.

Reply:
Yes, but the entry point sources the bashrc. I am confused.
We create the bashrc with conda-activate in it.
The entry point sources it.
Is that bashrc sourced there, potentially not the one we create in Dockerfile?

Reply:
Yeah sounds right but I don't quite recall the exact reason it was added there. Can we leave it in there for now and I can look into it and test it?

Reply:
Added back with comment.
When you have time check if it can be removed.
Not a big issue, but why leave it there if not needed.

Reply:
Agreed! Will do.